### PR TITLE
[zh] translate task/Install and Set Up kubectl on Windows

### LINKED ISSUE #27483: [zh] translate "tasks/Install and Set Up kubectl on Windows"
** This is a Feature Request **
translate content/docs/tasks/tools/install-kubectl-windows.md into Chinese

** What would you like to be added **
content/zh/docs/tasks/tools/install-kubectl-windows.md

/assign
/language zh

### PR DESCRIPTION
**Pull request by zhiguo-lu**
translate /docs/tasks/tools/install-kubectl-windows.md into Chinese.

close #27483

### CODE REVIEW

File: content/zh/docs/tasks/tools/install-kubectl-windows.md [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+1. 测试一下，确保安装的是最新版本：
+
+   ```powershell
+   kubectl version --client
+   ```
+
+   <!-- 
+   1. Navigate to your home directory:
+   -->
+1. 导航到家目录：
```
Comment (tengqm) [inline content/zh/docs/tasks/tools/install-kubectl-windows.md:218]:
```suggestion
1. 导航到 home 目录：
```

File: content/zh/docs/tasks/tools/install-kubectl-windows.md [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+
+   ```powershell
+   # If you're using cmd.exe, run: cd %USERPROFILE%
```
Comment (tengqm) [inline content/zh/docs/tasks/tools/install-kubectl-windows.md:221]: 221 可以翻译

File: content/zh/docs/tasks/tools/install-kubectl-windows.md [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+   kubectl version --client
+   ```
+
+<!-- 
+[Docker Desktop for Windows](https://docs.docker.com/docker-for-windows/#kubernetes) adds its own version of `kubectl` to `PATH`.
+If you have installed Docker Desktop before, you may need to place your `PATH` entry before the one added by the Docker Desktop installer or remove the Docker Desktop's `kubectl`.
+-->
+{{< note >}}
+[Windows 版的 Docker Desktop](https://docs.docker.com/docker-for-windows/#kubernetes) 将其自带版本的 `kubectl` 添加到 `PATH`。
+如果你之前安装过 docker Desktop，可能需要把此 `PATH` 条目置于 Docker Desktop 安装的条目之前，或者直接删掉 Docker Desktop 的 `kubectl`。
```
Comment (tengqm) [inline content/zh/docs/tasks/tools/install-kubectl-windows.md:131]:
```suggestion
如果你之前安装过 Docker Desktop，可能需要把此 `PATH` 条目置于 Docker Desktop 安装的条目之前，
或者直接删掉 Docker Desktop 的 `kubectl`。
```

File: content/zh/docs/tasks/tools/install-kubectl-windows.md [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+
+<!-- 
+### Install on Windows using Chocolatey or Scoop
+-->
+### 在 Windows 上用 Chocolatey 或 Scoop 安装 {#install-on-windows-using-chocolatey-or-scoop}
+
+<!-- 
+1. To install kubectl on Windows you can use either [Chocolatey](https://chocolatey.org) package manager or [Scoop](https://scoop.sh) command-line installer.
+-->
+1. 要在 Windows 上安装 kubectl，你可以使用包管理器 [Chocolatey](https://chocolatey.org) 或是 命令行安装器[Scoop](https://scoop.sh)。
```
Comment (tengqm) [inline content/zh/docs/tasks/tools/install-kubectl-windows.md:191]:
```suggestion
1. 要在 Windows 上安装 kubectl，你可以使用包管理器 [Chocolatey](https://chocolatey.org)
   或是命令行安装器 [Scoop](https://scoop.sh)。
```

Comment (tengqm) [reply content/zh/docs/tasks/tools/install-kubectl-windows.md:191]: 注意长行折行和空格

File: content/zh/docs/tasks/tools/install-kubectl-windows.md [hunk 5]
```diff
@@ -0,0 +1,295 @@
...
+### 启用 shell 自动补全功能 {#enable-shell-autocompletion}
+
+<!-- 
+kubectl provides autocompletion support for Bash and Zsh, which can save you a lot of typing.
+
+Below are the procedures to set up autocompletion for Zsh, if you are running that on Windows.
+-->
+kubectl 为 Bash 和 Zsh 提供自动补全功能，可以减轻许多输入的负担。
+
+下面是设置 Zsh 自动补全功能的操作步骤，前提是它在你的 Windows 上面已可以运行。
```
Comment (tengqm) [inline content/zh/docs/tasks/tools/install-kubectl-windows.md:289]:
```suggestion
下面是设置 Zsh 自动补全功能的操作步骤，前提是你在 Windows 上面运行的是 Zsh。
```

**Comment by zhiguo-lu** [comment]
https://deploy-preview-27567--kubernetes-io-master-staging.netlify.app/zh/docs/tasks/tools/install-kubectl-windows/

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**